packages/core/theme/src/components/button.ts

```diff
@@ -316,31 +319,36 @@ const button = tv({
       color: "danger",
       class: colorVariants.ghost.danger,
     },
-    // isInGroup / size
+    // isInGroup / radius
     {
       isInGroup: true,
-      size: "sm",
+      class: "rounded-none first:rounded-l-medium last:rounded-r-medium",
+    },
+    {
+      isInGroup: true,
+      radius: "sm",
       class: "rounded-none first:rounded-l-small last:rounded-r-small",
     },
     {
       isInGroup: true,
-      size: "md",
+      radius: "md",
       class: "rounded-none first:rounded-l-medium last:rounded-r-medium",
     },
     {
       isInGroup: true,
-      size: "lg",
+      radius: "lg",
       class: "rounded-none first:rounded-l-large last:rounded-r-large",
     },
     {
       isInGroup: true,
-      isRounded: true,
+      radius: "full",
       class: "rounded-none first:rounded-l-full last:rounded-r-full",
```

Comment:
But this isn't really related, is it, could we put this into a separate PR?

Reply:
Sure! I can recover the radius part for another PR.

Reply:
Solved it.